src/main/java/com/gr4vy/sdk/Gr4vyClient.java

```diff
@@ -167,13 +171,13 @@ public String getToken(String key, String[] scopes, Map<String, Object> embed, U
 	    JWSSigner signer = new ECDSASigner(e);
 	    
 	    Date now = new Date();
-	    Date expire = new Date(now.getTime() + 60 * 1000);
+	    Date expire = new Date(now.getTime() + tokenExpirySeconds * 1000);
```

Comment:
@steve-gr4vy any reason why we're setting this to seconds and multiplying instead of just expecting miliseconds?

Reply:
Hmm.. good point.. I'll update to Millis